<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="Frame_connexion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="boutonConnexion">
       <property name="text">
        <string>se connecter</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextBrowser" name="IndicConnexion">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>31</height>
        </size>
       </property>
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="batterie">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="camera">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>651</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QFrame" name="commandes">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>130</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="boutonStop">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/Image/stop.png</normaloff>:/Image/stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="boutonGauche">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/Image/Fleche gauche.png</normaloff>:/Image/Fleche gauche.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Left, Q</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="boutonHaut">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/Image/Fleche haut.png</normaloff>:/Image/Fleche haut.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Z, Up</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="boutonDroite">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/Image/Fleche droite.png</normaloff>:/Image/Fleche droite.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Right, D</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="boutonBas">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/Image/Fleche bas.png</normaloff>:/Image/Fleche bas.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Down, S</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="boutonPivotG">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/Image/PivotGauche.png</normaloff>:/Image/PivotGauche.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="shortcut">
        <string>A</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="boutonPivotD">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image.qrc">
         <normaloff>:/Image/PivotDroite.png</normaloff>:/Image/PivotDroite.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="shortcut">
        <string>E</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="capteurIr">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>350</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="Voiture">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>91</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="image.qrc">
       <normaloff>:/Image/voiture.png</normaloff>:/Image/voiture.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="shortcut">
      <string notr="true"/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="IndicDevantGauche">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>20</width>
       <height>30</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::BottomToTop</enum>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="IndicDevantDroite">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>20</width>
       <height>30</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::BottomToTop</enum>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="IndicDerriereDroite">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>170</y>
       <width>20</width>
       <height>30</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::BottomToTop</enum>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="IndicDerriereGauche">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>170</y>
       <width>20</width>
       <height>30</height>
      </rect>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::BottomToTop</enum>
     </property>
     <property name="format">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="odometrie">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>651</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QFrame" name="FrameVitesse">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>220</width>
      <height>85</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLCDNumber" name="Vitesse">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Filled</enum>
       </property>
       <property name="intValue" stdset="0">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextBrowser" name="kmParH">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>MS Sans Serif</family>
         <pointsize>12</pointsize>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Sans Serif'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;m/s&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QSlider" name="vitesse">
       <property name="minimum">
        <number>120</number>
       </property>
       <property name="maximum">
        <number>240</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionParametre"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionParametre">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/Image/pngegg.png</normaloff>:/Image/pngegg.png</iconset>
   </property>
   <property name="text">
    <string>Parametre</string>
   </property>
   <property name="toolTip">
    <string>Paremètre</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>boutonConnexion</tabstop>
  <tabstop>IndicConnexion</tabstop>
  <tabstop>vitesse</tabstop>
  <tabstop>kmParH</tabstop>
  <tabstop>boutonGauche</tabstop>
  <tabstop>boutonBas</tabstop>
  <tabstop>boutonDroite</tabstop>
  <tabstop>boutonHaut</tabstop>
  <tabstop>boutonStop</tabstop>
  <tabstop>Voiture</tabstop>
 </tabstops>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>vitesse</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>Vitesse</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>761</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>737</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
